tools/deployment/make_linux_package.sh

```diff
@@ -125,7 +126,12 @@ function main() {
     cp $OSQUERY_CONFIG_SRC $INSTALL_PREFIX/$OSQUERY_ETC_DIR/osquery.conf
   fi
 
-  if [[ $DISTRO = "centos7" || $DISTRO = "rhel7" || $DISTRO = "xenial" ]]; then
+  if [[ $DISTRO = "xenial" ]]; then
+  #Change config dir path for Xenial
+  sed -i 's/sysconfig/default/g' $SYSTEMD_SERVICE_SRC
+  fi
+
+  if [[ $DISTRO = "centos7" || $DISTRO = "rhel7" ]]; then
```

Comment:
Assuming you did want to change `DST` then you can keep the include for `xenial` and make the directories. Otherwise `xenial` will have the Upstart service installed via the `else`.